src/controllers/signupController.js

```diff
@@ -0,0 +1,43 @@
+import bcrypt from 'bcrypt';
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import SignupService from '../services/signupService';
+
+dotenv.config();
```

Comment:
You don't need to do this here again if it has already been declared in the entry point